eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -67,7 +67,7 @@ stages:
                         Get-ChildItem $(Pipeline.Workspace)/${{artifact.safeName}}
                       displayName: Stage artifacts
                       timeoutInMinutes: 5
-                    - template: eng/pipelines/templates/steps/create-tags-and-git-release.yml@azure-sdk-tools
+                    - template: ../../../common/pipelines/templates/steps/create-tags-and-git-release.yml
```

Comment:
While this likely works I think it would be nicer if we did it as a path from the root (assuming that works) like `eng/common/pipelines...`

Reply:
Absolute path did not work. it just broke the build.

Reply:
Oh that's a shame. Thanks for confirming though.